capstone/client/src/components/ListPage.js

```diff
@@ -60,8 +57,18 @@ class ListPage extends Component {
 
   componentWillMount = () => {
     this.selectedItems = new Set();
+    this.props.store.on('userId').subscribe(userId => {
```

Comment:
(userId) => {}

Always using parentheses even for single-parameter arrow functions can avoid situations where adding parameters, but forgetting to add parentheses.